doc/Command-Reference.md

```diff
@@ -3763,38 +3762,95 @@ This command displays all the mirror sessions that are configured.
+  ```
+
+  The following command is also supported to be backward compatible.
+  This command will be deprecated in future releases.
   ```
   config mirror_session add <session_name> <src_ip> <dst_ip> <dscp> <ttl> [gre_type] [queue]
   ```
 
 - Example:
   ```
-  admin@sonic:~$ sudo config mirror_session add mrr_abcd 1.2.3.4 20.21.22.23 8 100 0x6558 0
-  admin@sonic:~$ show mirror_session
-  Name       Status    SRC IP       DST IP       GRE     DSCP    TTL    Queue
-  ---------  --------  -----------  -----------  ------  ------  -----  -------
-  mrr_abcd   inactive  1.2.3.4      20.21.22.23  0x6558  8       100    0
+  root@T1-2:~# config mirror_session add mrr_legacy 1.2.3.4 20.21.22.23 8 100 0x6558 0
+  root@T1-2:~# show mirror_session
+  Name         Status    SRC IP     DST IP       GRE     DSCP    TTL    Queue    Policer    Monitor Port    SRC Port    Direction
+  ---------    --------  --------   -----------  ------  ------  -----  -------  ---------  --------------  ----------  -----------
+  mrr_legacy   inactive  1.2.3.4    20.21.22.23  0x6558  8       100    0
+
+
+  root@T1-2:~# config mirror_session erspan add mrr_abcd 1.2.3.4 20.21.22.23 8 100 0x6558 0
+  root@T1-2:~# show mirror_session
+  Name       Status    SRC IP     DST IP       GRE     DSCP    TTL    Queue    Policer    Monitor Port    SRC Port    Direction
+  ---------  --------  --------   -----------  ------  ------  -----  -------  ---------  --------------  ----------  -----------
+  mrr_abcd   inactive  1.2.3.4    20.21.22.23  0x6558  8       100    0
+  root@T1-2:~#
+
+  root@T1-2:~# config mirror_session erspan add mrr_port 1.2.3.4 20.21.22.23 8 100 0x6558 0 Ethernet0
+  root@T1-2:~# show mirror_session
+  Name       Status    SRC IP     DST IP       GRE     DSCP    TTL    Queue    Policer    Monitor Port    SRC Port    Direction
+  ---------  --------  --------   -----------  ------  ------  -----  -------  ---------  --------------  ----------  -----------
+  mrr_port   inactive  1.2.3.4    20.21.22.23  0x6558  8       100    0                                   Ethernet0   both
+  root@T1-2:~#
+  ```
+
+While adding a new SPAN session, users need to configure the following fields that are used while forwarding the mirrored packets.
+1) destination port,
+2) optional - List of source ports- List of source ports which can have both Ethernet and LAG ports.
+3) optional - Direction - Mirror session direction when configured along with Source port. (Supported rx/tx/both. default direction is both)
+4) optional - Queue in which packets shall be sent out of the device. Valid values 0 to 7 for most of the devices. Users need to know their device and the number of queues supported in that device.
+5) optional - Policer which will be used to control the rate at which frames are mirrored.
+
+- Usage:
+  ```
+  config mirror_session span add <session_name> <dst_port> [source-port-list] [direction] [queue] [policer <policer_name>]
```

Comment:
Does it uses ACL internally ?  Can we specify queue and policer without using ACL ? Also if does not use ACL internally can we police with direction as tx ?

Have we verified this combination and what is the behaviour at Sonic level if SAI return error for any of unsupported combination